actors/market/src/testing.rs

```diff
@@ -90,7 +90,8 @@ pub fn check_state_invariants<BS: Blockstore + Debug>(
     match DealArray::load(&state.proposals, store) {
         Ok(proposals) => {
             let ret = proposals.for_each(|deal_id, proposal| {
-                let proposal_cid = proposal.cid()?;
+                //let proposal_cid = proposal.cid()?;
+                let proposal_cid = todo!();
```

Comment:
Need the runtime to call `deal_cid`. We'll probably need to explicitly compute the blake2b hash here.